app/controllers/providers/application_merits_task/opponents_types_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Providers
+  module ApplicationMeritsTask
+    class OpponentsTypesController < ProviderBaseController
```

Comment:
**nit** why are we using a double plural here? would not `OpponentTypesController` make more sense. I know its inline with some other opponent related controllers but they too seem weird

Reply:
I renamed the `OpponentsNamesController` [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/5519/files#diff-3c6b193c4beb4d633c7dfa6c7f82b748737cc173f843207d66cbeaded946c877R3)

Reply:
👍 I had added a commit to rename the controller to be inline with other opponent related controllers but have now dropped this commit so that it is opponent_types_controller.rb